packages/neon_framework/lib/src/widgets/image.dart

```diff
@@ -235,10 +236,10 @@ class _NeonApiImageState extends State<NeonApiImage> {
     await RequestManager.instance.wrapBinary(
       account: widget.account,
       cacheKey: widget.cacheKey,
-      getCacheParameters: () async => CacheParameters(
-        etag: widget.etag,
-        expires: widget.expires,
-      ),
+      getCacheHeaders: () async => {
+        if (widget.etag != null) 'etag': widget.etag!,
+        if (widget.expires != null) 'expires': formatHttpDate(widget.expires!),
+      },
```

Comment:
I really hate lambdas pointing to a map as that can easily be read as a method body.
Please change it to a method body returning a map.